tests/store.test.js

```diff
@@ -71,18 +61,4 @@ describe('RedisStore', () => {
       expect(_del.mock.calls[0][0]).toBe(`authomatic_${userId}_${refreshToken}`);
     });
   });
-
-  describe('#removeAll', () => {
-    it('Should remove all refresh tokens for a single user', async () => {
-      store._scanKeys = jest.fn(() => ['1', '2', '3']);
```

Comment:
Maybe have `sancKeys` as an injected dependency, similar to how the redis client is injected.

Reply:
I added it as an issue